### PR TITLE
Docs/custom capability statement

### PR DESCRIPTION
**Pull request by aaburgos4**
@Robert-W let me know if I need to re-word anything or if something isn't entirely accurate. thanks!

### CODE REVIEW

**Comment by Robert-W** [comment]
Thanks for submitting this, we have added a few comments to this but overall this looks really good and thanks so much for submitting docs.

File: docs/CustomCapability.md [hunk 1]
```diff
@@ -1 +1,104 @@
...
+## Customizing the Capability Statement
+You now have the ability to generate a custom Capability Statement! Lets walk through how to do that. 
+
+### Creating a Statement Generator
+You can decide to include this code where you define your Asymmetrik Server configurations or include it in another file and simply require it in. For this purpose we will go with the latter option.
+
+First, we're going to require in the Asymmetrik FHIR Server source code. This will allow us to have access to the class that is responsible for generating the Capability Statement so that we can create our own.
+Then we'll create a function that will require in the CapabilityStatement class, and return a new instance that contains your information! Please make sure you remain compliant with the FHIR Specification.
+
+Next, we're going to create a custom Security Statement. This is apart of the Capability Statement. It provides information about the security of the current implementation.
```
Comment (sshah-asymmetrik) [inline docs/CustomCapability.md]: Can we change the phrase "This is apart of the Capability Statement" to "This is separate from the Capability Statement." for clarity? The first time time I read it I mis-interpreted it as "this is a part of."  Thanks!

File: docs/CustomCapability.md [hunk 2]
```diff
@@ -1 +1,104 @@
...
+  return {
+    makeStatement: customCapabilityStatement,
+    securityStatement: customSecurityStatement
+  };
+};
+```
+
+### Using the Statement Generator in the Config
+Now that we've built out our Statement Generator and have customized the Capability Statement info to our liking, we can add it to our configuration by doing the following: 
+```javascript
```
Comment (Robert-W) [inline docs/CustomCapability.md]: Just to be thorough and prevent any potential confusion, can we make this a full example and just add the following into it:

```javascript
const FHIRServer = require('@asymmetrik/node-fhir-server-core');

let config = {}; // Use the config and other things you provided here
let server = FHIRServer.initialize(config);
...
```

**Comment by aaburgos4** [comment]
@Robert-W @sshah-asymmetrik addressed the code review comments

**Comment by Robert-W** [comment]
Awesome, this looks great, thanks again.

**Pull request merged**